dask_deltatable/core.py

```diff
@@ -44,6 +48,10 @@ def __init__(
             path, storage_options=storage_options
         )
         self.schema = self.dt.schema().to_pyarrow()
+        meta = make_meta(self.schema.empty_table().to_pandas())
+        if self.columns:
+            meta = meta[self.columns]
+        self.meta = meta
```

Comment:
This is purely cosmetic. Since `self._make_meta_from_schema` was only used in one place I just moved it into `__init__` as an attribute